build-parent/pom.xml

```diff
@@ -252,7 +255,7 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <version>${version.surefire.plugin}</version>
           <configuration>
-            <argLine>-Xmx1024m -Dfile.encoding=UTF-8</argLine>
+            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 ${argLine}</argLine>
```

Comment:
this is a workaround as `<argLine>` for `surefire` plugin causes some issues to `jacoco` plugin, that is used to measure code coverage.

More details and solutions can be seen in this thread https://stackoverflow.com/questions/23190107/cannot-use-jacoco-jvm-args-and-surefire-jvm-args-together-in-maven

Reply:
Can you add this link as comment in the pom?